src/hooks/use-drag-and-drop-file.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef, useState } from 'react';
+
+export function useDragAndDropFile< T extends HTMLElement >( {
+	onFileDrop,
+}: {
+	onFileDrop: ( file: File ) => void;
+} ) {
+	const dropRef = useRef< T >( null );
+	const [ isDraggingOver, setIsDraggingOver ] = useState( false );
+
+	console.log( 'rendering', { isDraggingOver } );
```

Comment:
Do we need this console.log?

Reply:
You are right. Console logs are not needed, it was used just for debugging.